concent_api/core/tests/test_auth_accept_or_reject_results.py

```diff
@@ -183,13 +183,13 @@ def test_requestor_should_receive_subtask_results_from_concent_with_correct_keys
         # STEP 1: Provider forces subtask results via Concent.
         # Request is processed correctly.
         serialized_force_subtask_results = self._get_serialized_force_subtask_results(
-            timestamp                   = "2018-02-05 10:00:20",
+            timestamp                   = "2018-02-05 10:00:30",
```

Comment:
Please change the value to be the same as in line 51 - it will be easier in the future to find similar pieces.

Reply:
I've change line 51 to `10:00:30` so this one can stay as it is.